tests/conftest.py

```diff
@@ -78,3 +78,8 @@ def strategy(strategist, keeper, vault, Strategy, gov):
     strategy.setKeeper(keeper)
     vault.addStrategy(strategy, 10_000, 0, 2 ** 256 - 1, 1_000, {"from": gov})
     yield strategy
+
+
+@pytest.fixture
+def relative_aprox():
```

Comment:
Just make it capital so it's identified as a constant. Also, might as well session scope it.
```suggestion
@pytest.fixture(scope="session")
def RELATIVE_APPROX():
```

Reply:
ok, updated.